### DISCUSSION TITLE
OpenDTU MQTT Topics in Mosquitto on NAS

### DISCUSSION THREAD

**Discussion by masselmassel** [tbnobody/OpenDTU#866] [Q&A]
Hi there,
I would like to save the process of the daily earned energy of my HM-600 to a database and I want to show it in a graphic. 
Obviously this works with MQTT, but working with MQTT is new to me. 
I have running a Synology NAS. I have installed the Mosquitto Broker and I have made 3 folders "config" "log" and "data". There is a config-file:

"
pid_file /var/run/mosquitto.pid
persistence true
persistence_location /mqtt/data/
allow_anonymous true
user mosquitto
port 1883
log_dest file /mqtt/log/mosquitto.log
log_dest stdout
"

MQTT in the HM-600 is activated. The protocol in Mosquitto tells me, that there is a connection to the HM-600:
"1682923418: New client connected from 172.17.0.1 as OpenDTU-3650476 (c1, k15, u'BKW-broker')."

Where and how can I tell Mosquitto which of the topics it has to subscribe? Maybe someone can help. Thanks!
I tried to read and learn, but I can't get any further on my own at that point now.

**Comment by Kampfschildkroete** [comment]
Maybe I misunderstand your setup but MQTT is just a protocol to transfer data between devices. The management is done by the so called "broker" (in your case the "Mosquitto Broker"). By connecting the OpenDTU to the broker the DTU periodically sends data to the broker. The broker will send each data to all clients that are registered to the topic. 
What I understand from your setup is that you are missing the client that receives the data and writes it to the database?

**Reply by DPO99** [reply]
Correct! Should work with NodeRed and InfluxDB.
